gems/sorbet-runtime/lib/types/props/optional.rb

```diff
@@ -49,13 +49,13 @@ def add_prop_definition(prop, rules)
     if default_setter
       @props_with_defaults ||= {}
       @props_with_defaults[prop] = default_setter
-      @props_without_defaults&.delete(prop) # Handle potential override
+      props_without_defaults&.delete(prop) # Handle potential override
```

Comment:
ooc: why?

Reply:
There is already an `attr_reader` defined for this ivar and using it suppresses the  ivar initialization warnings. Is there a reason why we should not be using the reader method?